tests/app/core/models/github/github-label.model.spec.ts

```diff
@@ -0,0 +1,41 @@
+import { GithubLabel } from '../../../../../src/app/core/models/github/github-label.model';
+
+describe('GithubLabel', () => {
+  let githubLabel: GithubLabel;
+  const category = 'severity';
+  const value = 'LOW';
+
+  it('should returns false on isCategorical() when provided wit no name', () => {
```

Comment:
Minor nitpick: let's phrase these descriptions as `.isCategorical() should return <bool> when ....`

Reply:
Eyes! :+1: